### ISSUE TITLE
Ask some questions

### ISSUE THREAD

**Issue by CoderBinGe** [mswellhao/PacSum#15]
**I have some questions for advice, looking forward to your reply：**
1. the finetuned bert models you give ，is based on data used in the paper?
2. How to train my own data, is there anything I can refer to? you didn't include the training code in this repo
3. No matter it is the training set or the test set, there is an abstract field. Shouldn't the abstract of the test set be unknown?
